runtime/src/cli.rs

```diff
@@ -73,7 +73,10 @@ pub fn parse_duration_minutes(src: &str) -> Result<Duration, ParseIntError> {
 #[derive(Parser, Debug, Clone)]
 pub struct ConnectionOpts {
     /// Parachain websocket URL.
-    #[clap(long, default_value = "ws://127.0.0.1:9944")]
+    #[cfg_attr(feature = "standalone-metadata",clap(long, default_value = "ws://127.0.0.1:9944"))]
+    #[cfg_attr(feature = "parachain-metadata-testnet",clap(long, default_value = "wss://api-testnet.interlay.io:443/parachain"))]
+    #[cfg_attr(feature = "parachain-metadata-kintsugi",clap(long, default_value = "wss://api-kusama.interlay.io:443/parachain"))]
+    #[cfg_attr(feature = "parachain-metadata-interlay",clap(long, default_value = "ws://127.0.0.1:9944"))]
```

Comment:
```suggestion
    #[cfg_attr(feature = "parachain-metadata-interlay",clap(long, default_value = "wss://api.interlay.io/parachain"))]
```

Good change! Let's set up the interlay url as well